docs/lake/design.md

```diff
@@ -672,8 +642,8 @@ sort of the KSUIDs results in a creation-time ordering (though this ordering
 is not relied on for causal relationships since clock skew can violate
 such an assumption).
 
-Data objects are referred to by zero or more commits, where the commits
-are maintained in a commit journal described below.
+Data objects are referred to by zero or more commit objects, where the =
+commits are maintained referenced by the branches journal described below.
```

Comment:
I'm not sure what suggestion to make to fix this sentence, but "maintained referenced by" reads funny.

Reply:
deleting